website/thaliawebsite/templates/index.html

```diff
@@ -14,6 +14,11 @@
 {% block body %}
     <section id="thaliawebsite-frontpage" class="page-section">
         <div class="container">
+            {% if messages %}
```

Comment:
This should be in a separate PR if we want to do it properly since it closes #1616

Reply:
Do we really want to do it properly, or shall I just close 1616 with this 😅

Reply:
Well that is what I was asking you.